charts/telegraf/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.serviceMonitor.create }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
```

Comment:
Be aware of situation that Prometheus resources can be installed in different namespace than Telegraf: By default and before the version v0.19.0, ServiceMonitors must be installed in the same namespace as the Prometheus resource. With the Prometheus Operator v0.19.0 and above, ServiceMonitors can be selected outside the Prometheus namespace via the serviceMonitorNamespaceSelector field of the Prometheus resource. The discovered targets may come from any namespace. This allows cross-namespace monitoring use cases, for example, for meta-monitoring.

Reply:
Done